Automattic/newspack-blocks
# Feat/donate block tiers layout

### All Submissions:






### Changes proposed in this Pull Request:



Adds an alternative, "tiers-based" layout to the Donate block: 

<img width="811" alt="image" src="https://user-images.githubusercontent.com/7383192/202211905-53f18f58-ea99-491f-a292-2fa6515e2cbf.png">

_Above in the "Alternate" block style version_

It's called "Tiers-based" layout (vs. the default "Frequency-based" layout), but I'm not sure about the naming here. Suggestions are welcome.

<img width="287" alt="image" src="https://user-images.githubusercontent.com/7383192/202214354-37128929-17a7-40fc-b374-ec418eeb0f9c.png">

Closes https://github.com/Automattic/newspack-product/issues/53

### How to test the changes in this Pull Request:

1. Set Stripe as the Reader Revenue platform
1. On `master`, create a page with three Donate blocks, set each to different style
2. Switch to this branch, ensure these all render as expected
3. Switch all to "Tiers-first" in the sidebar "Layout" panel
4. Visit the front-end, make a few donations using the new block layout, observe amounts and frequencies translating to Stripe 

### Other information:
